pkg/client/clientset/test/testgroup_test.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test_test
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"k8s.io/kubernetes/cmd/libs/go2idl/client-gen/testdata/apis/testgroup"
+	_ "k8s.io/kubernetes/cmd/libs/go2idl/client-gen/testdata/apis/testgroup/install"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/latest"
+	"k8s.io/kubernetes/pkg/api/testapi"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	. "k8s.io/kubernetes/pkg/client/clientset/test"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/client/unversioned/testclient/simple"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+var testHelper testapi.TestGroup
+
+func init() {
+	if _, found := testapi.Groups[testgroup.SchemeGroupVersion.Group]; found {
+		return
+	}
+	testapi.Groups[testgroup.SchemeGroupVersion.Group] = testapi.NewTestGroup(
```

Comment:
I have to register this group to `testapi` because simple_testclient.go accesses this this group via `testapi`.